README.md

```diff
@@ -191,7 +192,8 @@ While some implicit technical details are obvious to you, they might not be for
 ## Maintainability
 
 * Find the right balance between being general and being precise (aka Captain Obvious)
-* Very detailed step-by-step articles are easy to follow but get outdated very quickly 
+* Very detailed step-by-step articles are easy to follow but get outdated very quickly
```

Comment:
This is another reason to keep the explanation separate from the code. First explain the process in human terms, then provide concrete example with commands, which may get outdated soon.